A90-health-service-list-method.md

```diff
@@ -24,21 +24,20 @@ Kubernetes provides a similar capability with its `/readyz?verbose` endpoint, wh
 
 ### Related Proposals
 
-N/A
+- [A17 - Client-Side Health Checking](https://github.com/grpc/proposal/blob/master/A17-client-side-health-checking.md)
```

Comment:
This can be a relative link, since it's the same directory as the thing it's pointing to -- i.e., the link target just needs to be "A17-client-side-health-checking.md".

Reply:
You're right on all accounts, I forgot about relative references in markdown for some reason, thank you!

Reply:
```suggestion
- [A17 - Client-Side Health Checking](A17-client-side-health-checking.md)
```